<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/logoPrefix/pirctures/logo.png</normaloff>:/logoPrefix/pirctures/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(191, 194, 198)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="logo.qrc">:/logoPrefix/pirctures/lock.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_register">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(116, 253, 116)
</string>
          </property>
          <property name="text">
           <string>Зарегистрироваться</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 123, 123)</string>
          </property>
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Фамилия:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_secondName">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Отчество:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_thirdName">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Логин:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_login">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Эл. почта:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_email">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Пароль:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Повторите пароль:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_repeatedPassword">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Имя:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_firstName">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Registration</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_register</sender>
   <signal>clicked()</signal>
   <receiver>Registration</receiver>
   <slot>registrate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>registrate()</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
